File: Course_code_app/BE_app/Application/DTOs/Lesson.cs
namespace Application.DTOs;

public class LessonDto
{
    public int Id { get; set; }
    public int CourseId { get; set; }
    public string Title { get; set; } = string.Empty;
    public string? ContentSummary { get; set; } 
    public int Position { get; set; }
    public string Status { get; set; } = "Chưa học";
    public DateTime CreatedAt { get; set; }
    public DateTime? UpdatedAt { get; set; }
}

File: Course_code_app/FE_app/lib/Models/lesson.dart
class Lesson {
  final int id;
  final int? courseId;
  final String title;
  final String? contentSummary;
  final int? position;
  final String status; // 'Chưa học', 'Đang học', 'Đã hoàn thành'
  final DateTime? createdAt;
  final DateTime? updatedAt;

  Lesson({
    required this.id,
    this.courseId,
    required this.title,
    this.contentSummary,
    this.position,
    required this.status,
    this.createdAt,
    this.updatedAt,
  });
}